src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star is-active"></div>
```

Comment:
You shouldn't any additional classes to `stars__star` elements. Consider using stars modifier to set the correct styles
![image](https://github.com/mate-academy/layout_stars/assets/96409563/047a6a8c-39f6-45c5-a1d0-d1e13e82c46a)